src/pages/Main/PostList/PostList.js

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components';
+import { setFlex, theme } from '../../../styles/theme';
+function Post(post) {
+  return (
+    <PostCard>
+      <PostImageWrap>
+        <img alt="company" src={post.post.img} />
+      </PostImageWrap>
+      <CompanySummery>
+        <h2>{post.post.title}</h2>
+        <p>{post.post.company}</p>
+        <p className="pTagGray">{post.post.location}</p>
+        <p className="pTagGray">{post.post.reward}</p>
+      </CompanySummery>
+    </PostCard>
+  );
+}
+
+function PostList(posts) {
+  return (
+    <PostCaradList>
+      {posts.posts.map((post, index) => (
+        <Post post={post} key={index} />
+      ))}
+    </PostCaradList>
+  );
+}
+
+export default PostList;
+
+const PostCaradList = styled.ul`
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  row-gap: 10px;
+  column-gap: 10px;
+`;
+
+const PostCard = styled.li`
+  height: 334px;
+  width: 270px;
+  overflow: hidden;
+  ${setFlex()};
+  flex-direction: column;
+`;
+
+const PostImageWrap = styled.div`
+  & img {
```

Comment:
```suggestion
const PostImage = styled.img`
```
img 태그에만 스타일을 줄거라면 바깥쪽 요소는 그냥 div 태그로 놓고 img 태그를 스타일드 컴포넌트로 만들면 되지 않을까요?

Reply:
처음에 쓴 코드들이 완전 이해를 못하고했었내요 ㅠㅠ 확인하고 못올려서 죄송합니다.